justfile

```diff
@@ -86,15 +86,17 @@ devenv: virtualenv
         $PIP install --no-deps -r "$req_file"
         [[ -v CI ]]  && echo "::endgroup::" || true
 
-        # Ensure playwright and chromium is installed
-        playwright install --with-deps chromium
-
         # Make a record of what we just installed
         cp "$req_file" "$record_file"
       fi
     done
 
 
+# install playwright; use --with-deps to include system dependencies
+playwright-install *ARGS: devenv
+    $BIN/playwright install {{ ARGS }}
```

Comment:
The reason I wrote this just command was so that CI could use --with-deps
I had an idea that it would have to be Edge in Level 4; if it can be chrome, then yes, chromium only would be fine

Reply:
Will give the fixture a try :+1: